### ISSUE TITLE
The runtime<->aspnetcore shared src is out of sync

### ISSUE THREAD

**Issue by Tratcher** [dotnet/aspnetcore#18943]
This is a tracking issue for when the [shared src](https://github.com/dotnet/aspnetcore/tree/master/src/Shared/runtime) gets out of sync with it's runtime counterpart. See the [ReadMe](https://github.com/dotnet/aspnetcore/blob/master/src/Shared/runtime/ReadMe.SharedCode.md) for details.

There is a github action that runs daily at https://github.com/dotnet/aspnetcore/actions and will re-open and comment on this bug if it detects any differences in the files shared between the repos. AspNetCore will triage and assign this issue to get the differences resolved.

**Comment by anurse** [comment]
We are currently in-sync.

**Comment by anurse** [comment]
Should be in sync now.

**Comment by Tratcher** [comment]
Fixed by https://github.com/dotnet/runtime/pull/32944

**Comment by campersau** [comment]
GitHub actions also run in the forked repositories. See https://github.com/campersau/AspNetCore/actions

You can prevent that by checking for the repository name in the workflow:
https://github.community/t5/GitHub-Actions/Do-not-run-cron-workflows-in-forks/m-p/46787/highlight/true#M6612

**Comment by Tratcher** [comment]
@anurse The Action failed. github-actions was able to re-open the issue but not comment, probably because the issue was locked. Any way to prevent it from getting locked?

**Comment by anurse** [comment]
Yep, we can update the rule to avoid locking it, maybe based on a label (so we can apply it in other scenarios). I’ll look at that later today. You should have permissions to unlock it for now, and any comment should reset the lock clock to 90 days.

**Comment by anurse** [comment]
Dorp, we’re on the issue now so of course you’ve already unlocked it :P

**Comment by adityamandaleeka** [comment]
Currently blocked by SDK update.

**Comment by Tratcher** [comment]
https://github.com/dotnet/runtime/commit/5584b26f1a0dc60406a5a51b0e4da14268c2ba1b, needs porting.
The script didn't open a PR for some reason...

**Comment by amcasey** [comment]
https://github.com/dotnet/runtime/pull/105765
